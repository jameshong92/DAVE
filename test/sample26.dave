// test table access
int main() {
    rec r1 = new rec(age: 22, name:"Yang", GPA:3.50); 
    rec r2 = new rec(name:"James", age:23, GPA:3.88);
    rec r3 = new rec(age:24, GPA:3.80, name:"Min Woo");
    rec r4 = [r1,r2,r3];
    tbl test = new tbl(r4);
    print(test);
    rec test1 = access(test, 2);
    print(test1);
}