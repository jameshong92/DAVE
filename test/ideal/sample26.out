age	name	GPA	
22	Yang	3.5	
23	James	3.88	
24	Min Woo	3.8	

age	name	GPA	
24	Min Woo	3.8	

